content/docs/guide/map_submission/map_zoning.md

```diff
@@ -9,60 +9,420 @@ weight: 4
+**Example: surf_tuscany zones**
+![Staged Surf Zones](/images/map_zoning/zoning_surf_staged.png)
+
+### Surf: Staged-Linear
+Staged-Linear Surf maps should be zoned **mostly** the same way as [Staged Surf](/guide/map_submission/map_zoning/#surf-staged) maps.  
+The only exception being that **Limit Ground Speed** needs to be unchecked for **every stage**.  
+**Example: surf_anzchamps zones**
+![Staged-Linear Surf Zones](/images/map_zoning/zoning_surf_staged_linear.png)
+
+### Rocket / Sticky Jump: No Courses
+Rocket / Sticky Jump maps without any **courses** should have **1 segment** and a **checkpoint** at the start of **every jump**.  
+**Example: rj_summer zones**
+![RJ/SJ No Courses Zones](/images/map_zoning/zoning_rocket_sticky_jump_no_courses.png)
+
+### Rocket / Sticky Jump: With Courses
+Rocket / Sticky Jump maps usually teleport the player back to the hub upon completion of a **course**.  
+{{<hint warning>}}
+
+This guide applies only to maps that require courses to end **before** the start of next course.  
+If the next course starts at the same spot where the previous course ends, **don't change the main track property** and **don't add checkpoints** to the end of **courses**.   
+If you have any questions please ask in **#map-porting** channel on our [Discord](https://discord.gg/momentummod).
+
+{{</hint>}}
+
+1. Uncheck **End segment at next segment start** in **main track** properties
+    - This will make the time submit when the player enters **last checkpoint** on a **course** except for the **last course**
+    - **Last course** time will **always** submit when player enters the [End Zone](/guide/map_submission/map_zoning/#creating-the-end-zone)
+![RJ/SJ Courses Properties](/images/map_zoning/zoning_rocket_sticky_jump_courses_properties.png)
+2. For every **course except last**:
+    1. Create a **new segment** at the first jump
+    2. Add a **checkpoint** to the start of every jump
+    3. Add a **checkpoint** to the end platform ( before being teleported to the hub )
+3. For the **last course**:
+    1. Create a **new segment** at the first jump
+    2. Add a **checkpoint** to the start of every jump
+        - **Do not** add a checkpoint to the end of this course
+
+**Example: rj_deserted zones** ( notice that the end of **course 1** is zoned with a **checkpoint** )
+![RJ/SJ Courses](/images/map_zoning/zoning_rocket_sticky_jump_courses.png)
+
+### Other Gamemodes
+Vast majority of maps for gamemodes other than **Surf** and **Rocket / Sticky Jump** will use a **single stage** with **checkpoints**.  
+That means they should be zoned identically to [Surf: Linear](/guide/map_submission/map_zoning/#surf-linear) maps.  
+
+{{<hint info >}}
+
+The only difference is that for modes other than Surf the **Limit ground speed** checkbox will be automatically **unchecked**.  
+That is the correct behavior so you don't have to worry about it.
```

Comment:
This setting really only applies to segments after the first, so I don't think it needs to be mentioned here.